lib/vanity/adapters/redis_adapter.rb

```diff
@@ -209,6 +213,12 @@ def destroy_experiment(experiment)
         @experiments.del *alternatives unless alternatives.empty?
       end
 
+      def get_saw_variation_time(experiment, identity)
```

Comment:
I'm not sure if there's a strict policy about maintaining consistency between the adapters, but it'd probably be good to add support for this to the other ones as well.